fsspec/implementations/tests/test_dbfs.py

```diff
@@ -150,10 +150,145 @@ def test_dbfs_read_range(dbfsFS):
 def test_dbfs_read_range_chunked(dbfsFS):
     dbfsFS.rm("/FileStore/large_file.txt")
     assert "/FileStore/large_file.txt" not in dbfsFS.ls("/FileStore/", detail=False)
-    content = b"This is a test\n" * (1 * 2**18) + b"For this is the end\n"
+    content = b"This is a test\n" * (1 * 2 ** 18) + b"For this is the end\n"
     with dbfsFS.open("/FileStore/large_file.txt", "wb") as f:
         f.write(content)
     assert "/FileStore/large_file.txt" in dbfsFS.ls("/FileStore", detail=False)
     assert dbfsFS.cat_file("/FileStore/large_file.txt", start=8) == content[8:]
     dbfsFS.rm("/FileStore/large_file.txt")
     assert "/FileStore/large_file.txt" not in dbfsFS.ls("/FileStore/", detail=False)
+
+
+@pytest.mark.vcr()
```

Comment:
The recordings were not included.

Reply:
I have recorded the tests and uploaded the new cassets